astro/src/content/docs/release-notes/index.mdx

```diff
@@ -27,6 +31,9 @@ Looking for release notes older than 1.23.0? Look in the [release notes archive]
   Removal of this field is targeted for the end of 2024.
 </DeprecationWarning>
 
+### New
+* Version `1.52.0` introduces support for Licensed Community. After upgrading, those with a community plan can now get even more out of FusionAuth with a free license.
```

Comment:
Feels like we perhaps should have a link to a blog post here for users to get more information on Licensed Community.

Reply:
Better to have 'more information' as part of the docs. I don't think we push folks from docs -> blog except in the narrow 'related blog posts' case.